Jenkinsfile

```diff
@@ -0,0 +1,16 @@
+properties([])
+node(){
+  deleteDir()
+  stage("Prepare"){
+    checkout scm
+    lint_container = docker.build 'lint'
+  }
+  lint_container.inside {
```

Comment:
That message is odd, I think its a misclassified warning that docker-in-docker won't work because the job actually does work. https://github.com/jenkinsci/docker-workflow-plugin/blob/68c1fd628f0918974b460bdc55598834a281093b/src/main/java/org/jenkinsci/plugins/docker/workflow/WithContainerStep.java#L141